### PR TITLE
Allow column reordering

### LINKED ISSUE #22: Reordering columns
In [flexboxgrid.css](http://flexboxgrid.com/) reordering props like `first` or `last` are concerned to use with Cols. Here it can be used only with [rows](react-flexbox-grid/src/components/Row.js#L42).

``` jsx
Row.propTypes = {
  reverse: PropTypes.bool,
  start: ModificatorType,
  center: ModificatorType,
  end: ModificatorType,
  top: ModificatorType,
  middle: ModificatorType,
  bottom: ModificatorType,
  around: ModificatorType,
  between: ModificatorType,
  first: ModificatorType,
  last: ModificatorType,
  className: PropTypes.string,
  tagName: PropTypes.string,
  children: PropTypes.node
};
```

In this case I can't do something like that:

``` html
<div class="row">
    <div class="col-xs-2">
        <div class="box">
            1
        </div>
    </div>
    <div class="col-xs-2">
        <div class="box">
            2
        </div>
    </div>
    <div class="col-xs-2 first-xs">
        <div class="box">
            3
        </div>
    </div>
</div>
```

### PR DESCRIPTION
**Pull request by eemeli**
This fixes #22 by moving the `first` and `last` props from `Row` to `Col`, as they are in `flexboxgrid.css`.

To verify that they work together, I've a build [here](/eemeli/react-flexbox-grid/tree/74f2ac5) that has this and PR #66 both merged, and includes the `lib/` build artifacts.

### CODE REVIEW

**Comment by madvas** [comment]
what's status of this?

**Comment by eemeli** [comment]
I at least have heard nothing since submitting the PR, and have in fact been considering actively forking this repo in order to make my own life easier.

@roylee0704, are you still actively developing this project?

**Comment by silvenon** [comment]
@eemeli hey, I'm sorry that this PR has been sitting here for 4 months, that's really too long! Personally I started using styled-components, which didn't jive with this library anymore, so I wasn't using it anymore.

I'll review this PR today, thanks a lot!

File: src/components/Col.js [hunk 1]
```diff
@@ -1,18 +1,19 @@
+import style from 'flexboxgrid';
```
Comment (silvenon) [inline src/components/Col.js]: Any reason why was this moved to the top?

File: test/components/Col.spec.js [hunk 2]
```diff
@@ -17,6 +17,11 @@ describe('Col', () => {
     expect(className).toContain(style['col-lg-4']);
   });
 
+  it('Should add "first-*" class if "first" property is set', () => {
+    renderer.render(<Col first='md'/>);
```
Comment (silvenon) [inline test/components/Col.spec.js]: Use double quotes for prop values (`<Col first="md" />`), I think this is the only reason why tests fail.

File: src/components/Col.js (hunk 1 above)
Comment (eemeli) [inline src/components/Col.js]: I find it' usually clearer to group the external imports separate from the internal ones, that's all.

**Comment by pokonski** [comment]
@silvenon do you know when we can expect a release to NPM with those changes?

**Comment by silvenon** [comment]
@pokonski not sure yet. The problem is that I discovered some bugs (i.e. some new features don't work) which I need to fix first and add more tests, but it shouldn't take long. Also, I don't think I have releasing powers, so the release time is not entirely up to me. 😕 

But I respond to mentions, so feel free to ping me occasionally, I like healthy pressure. 😉

**Comment by pokonski** [comment]
Sure, no worries. Just wanted to know when to expect it, cheers!

**Comment by silvenon** [comment]
Hopefully by the end of this week. 😉

**Comment by vemundeldegard** [comment]
What happened to this?

**Comment by silvenon** [comment]
@vemundeldegard check if there have been any releases since 13 Feb 2017 (that's when this was merged). If there have, it's released. Probably not documented, though.
